provider/aws/aws.go

```diff
@@ -120,7 +120,7 @@ func (p *AWSProvider) config() *aws.Config {
 		config.Endpoint = aws.String(p.Endpoint)
 	}
 
-	if os.Getenv("DEBUG") == "true" {
+	if os.Getenv("DEBUG") == "true" || os.Getenv("CONVOX_DEBUG") == "true" {
```

Comment:
`CONVOX_DEBUG` it is. Lets remove it from this line though as it won't have any affect in a rack and could lead to confusion. We could in another PR change this throughout the rack.